dags/common/constants.py

```diff
@@ -2,3 +2,4 @@
 
 ARXIV_EXTRACTION_PATTERN = re.compile(r"(arxiv:|v[0-9]$)", flags=re.I)
 NODE_ATTRIBUTE_NOT_FOUND_ERRORS = (AttributeError, TypeError)
+WHITE_SPACES = re.compile(r"[\n\t]{0,}" + r"\s{2,}")
```

Comment:
why it can't be one string?

Reply:
+ It is "AND" operator:
https://blog.finxter.com/python-regex-and-operator-tutorial-video/